<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Base</class>
 <widget class="QDialog" name="Base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>541</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>82</width>
     <height>59</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add row</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Delete row</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>653</x>
     <y>51</y>
     <width>82</width>
     <height>59</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Patients</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Doctors</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>tableView</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
